saq/remediation/mail/__init__.py

```diff
@@ -102,7 +102,7 @@ def failed_remediation(remediation: database.Remediation, remediator: BaseRemedi
 
 
 class EmailRemediationSystem(RemediationSystem):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, company_id=None, *args, **kwargs):
```

Comment:
Should the email remediation system be the only system that takes company_id into account? Or should they all work like that. I know there is only one remediation system ATM but that will change in the future.

Reply:
No it shouldn't I made company_id an optional argument for the RemediationSystem class so future remediation systems can be customized as needed.